tests/test_python/test_retrieve_data.py

```diff
@@ -481,15 +481,79 @@ def test_ufs_lbcs_from_aws(self):
                 '--config', self.config,
                 '--cycle_date', '2020072300',
                 '--data_stores', 'aws',
-                '--external_model', 'UFS-CASE-STUDY',
+                '--data_type', 'UFS-CASE-STUDY',
                 '--fcst_hrs', '3', '6', '3',
                 '--output_path', tmp_dir,
                 '--ics_or_lbcs', 'LBCS',
                 '--debug',
-                '--file_type', 'nemsio',
+                '--file_fmt', 'nemsio',
                 '--check_file',
             ]
             # fmt: on
 
             # Testing that there is no failure
             retrieve_data.main(args)
+
+            # Verify files exist in temp dir
+
+            path = os.path.join(tmp_dir, "*")
+            files_on_disk = glob.glob(path)
+            self.assertEqual(len(files_on_disk), 2)
```

Comment:
@mkavulich - It looks like these lines are causing the Python Functional Tests to fail.  Since this test is using the new --check_file flag, no files are being downloaded, so the verification that files exist in temp dir will fail, leading to the failure in the Python Functional Tests.

Reply:
This has been resolved with hash [ae8ebc8](https://github.com/ufs-community/ufs-srweather-app/pull/778/commits/ae8ebc88c1f736b05231678610b53c87834b0c01).  Resolving now.